cmd/buildah/rmi.go

```diff
@@ -27,7 +27,7 @@ var (
 		},
 		cli.BoolFlag{
 			Name:  "force, f",
-			Usage: "force removal of the image",
+			Usage: "force removal of the image and the containers using the image",
```

Comment:
nit 
`and any containers using the image.

Reply:
fixed, new patch inc.